apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -192,4 +196,141 @@ describe('Ticket:permissions', () => {
             await updateTestTicket(client, obj.id, { property: { connect: { id: client2.property.id } } })
         })
     })
+
+    test('employee from "from" organization: can read tickets from "to" organizations', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const { clientFrom, organizationTo, propertyTo, organizationFrom } = await createTestOrganizationLinkWithTwoOrganizations()
+        const clientTo2 = await makeClientWithProperty()
```

Comment:
Why are you creating extra client and calling `createTestOrganizationLink` below, when it seems, like `createTestOrganizationLinkWithTwoOrganizations` does it all?

Reply:
`createTestOrganizationLinkWithTwoOrganizations` creates a link between 2 organizations and returns them. In this test, I wanted the `organizationFrom` to have 2 related "to" organizations, and so I created another relationship between `organizationFrom` and `clientTo2.organization`